src/clj/webchange/common/audio_parser/recognizer.clj

```diff
@@ -0,0 +1,77 @@
+(ns webchange.common.audio-parser.recognizer
+  (:require
+    [clojure.data.json :as json]
+    [clojure.string :as s]
+    [clojure.java.io :as io]
+    [clojure.java.shell :refer [sh]]))
+
+(def parser-directory "src/clj/webchange/common/audio_parser/rhubarb/")
+
+(defn relative->absolute-path
+  [relative-path]
+  (str "resources/public" relative-path))
```

Comment:
Let's make 'public' folder absolute path available through 'env' so that it can be configured for each environment
See webchange/db/core.clj:21 for example

resources folder is not available when you start the server via uberjar